memory_test.go

```diff
@@ -613,64 +645,62 @@ func TestMemObjectsWithPropsLenThreshold(t *testing.T) {
 		{
 			"object under threshold",
 			`y = {}
-				checkMem();
-				for (i=0;i<10;i++) {
-					y["i"+i] = i
-				}
-				checkMem()`,
+			let i =0;
+			checkMem();
+			for (i=0;i<10;i++) {
+				y["i"+i] = i
+			}
+			checkMem()`,
 			100,
 			memUsageLimit,
-			// len("i0") + value i
-			10*2 + 10*SizeNumber +
-				// for loop 0 + 10 + len("i")
-				SizeNumber + SizeNumber + 1,
+			// object overhead + len("i0") + value i
+			SizeEmpty + 10*(2+SizeString) + 10*SizeNumber,
 		},
 		{
 			"object under threshold but over limit",
 			`y = {}
-				checkMem();
-				for (i=0;i<10;i++) {
-					y["i"+i] = i
-				}
-				checkMem()`,
+			let i = 0;
+			checkMem();
+			for (i=0;i<10;i++) {
+				y["i"+i] = i
+			}
+			checkMem()`,
 			100,
 			40,
-			// len("i0") + value i
-			10*2 + 10*SizeNumber +
-				// for loop 0 + 10 + len("i")
-				SizeNumber + SizeNumber + 1,
+			// object overhead + len("i0") + value i
+			SizeEmpty + 10*(2+SizeString) + 10*SizeNumber,
 		},
 		{
 			"object over threshold",
 			`y = {}
+			let i = 0;
 			checkMem();
 			for (i=100;i<200;i++) {
 				y["i"+i] = i
 			}
 			checkMem()`,
 			75,
 			memUsageLimit,
-			// len("i100") + value i
-			100*4 + 100*SizeNumber +
-				// for loop 100 + 200 + len("i")
-				SizeNumber + SizeNumber + 1,
+			// object overhead + len("i100") + value i
+			SizeEmpty + 100*(4+SizeString) + 100*SizeNumber,
 		},
 		{
 			"mixed object over threshold",
 			`y = {}
+			let i = 0;
 			checkMem();
 			for (i=100;i<200;i++) {
 				y["i"+i] = i < 150 ? i : "i"+i
 			}
 			checkMem()`,
 			200,
 			memUsageLimit,
-			// len("i100") + value i
-			50*4 + 50*SizeNumber +
-				// len("i150") + len("i150") + 150 number + len("i")
-				50*4 + 50*4 + 3 + 1 +
-				// for loop 100 + 200 + len("i")
-				SizeNumber + SizeNumber + 1,
+			// object overhead + key len("i100") + value i for the items 100 to 149
+			SizeEmpty + 50*(4+SizeString) + 50*SizeNumber +
```

Comment:
[q] The question is not related to your change. But do you know why the multiplier is `50`? Thanks.

Reply:
for the first 50 entries the value is just `i` but after that it is `"i"+i`